source/_integrations/manual.markdown

```diff
@@ -1,5 +1,5 @@
 ---
-title: Manual
+title: Manual Alarm Control Panel
```

Comment:
This has no effect, as the title is synced from the codebase.

Reply:
Ah. Good to know, thanks.